### PR TITLE
Increase pixi resolution option by default for a Creation template

### PR DESCRIPTION
**Pull request by CyberDex**

### CODE REVIEW

File: templates/template-creation-web/src/engine/engine.ts [hunk 1]
```diff
@@ -36,6 +37,9 @@ export class CreationEngine extends Application {
   /** Initialize the application */
   public async init(opts: Partial<ApplicationOptions>): Promise<void> {
     opts.resizeTo ??= window;
+    opts.resolution ??= getResolution();
+    opts.antialias ??= true;
```
Comment (Zyie) [inline templates/template-creation-web/src/engine/engine.ts:41]: While i agree that turning anti aliasing on makes the template look better, my only reservation here is that this option impacts performance a lot, especially for mobile devices, and I wouldn't want new users falling into this trap

I'll have a think about this one and get back to you

Comment (CyberDex) [reply templates/template-creation-web/src/engine/engine.ts:41]: We can skip `antialias`, the resolution here is what makes difference the most

Comment (Zyie) [reply templates/template-creation-web/src/engine/engine.ts:41]: Yeah let's skip`antialias` for now

Comment (CyberDex) [reply templates/template-creation-web/src/engine/engine.ts]: Removed

**Review by Zyie — APPROVED** [review]

**Pull request merged**